suites/test_new_draft.py

```diff
@@ -0,0 +1,127 @@
+        display_module_page.add_metadata_and_publish(self.driver)
+        utilities.wait_for_element(self.driver, By.ID, locators.MODULE_DISPLAY_UNPUBLISH_BUTTON_ID)
+
+
+    @lcc.test("Upload the repo via uploader script 2nd time")
+    def upload_repo_again(self):
+        lcc.log_info("Uploaded the repo again")
+        os.chdir(fixture.project_dir_git)
+        repo = Repo(fixture.project_dir_git)
+        git = repo.git
+        git.checkout("at-uploader2")
+        subprocess.check_call(
+                ('pantheon --user={} --password={} --server={} push'.format(uploader_username, uploader_password,
+                                                                            url)), shell=True)
+        # search_page.search_for_module_and_click(self.driver, constants.published_module)
+        search_beta_page.select_repo(self.driver, fixture.repo_name)
+        search_beta_page.search_module_and_click(self.driver, constants.assembly_new_draft_version+" 2")
+        check_that("Draft card is displayed",
+                   utilities.find_element(self.driver, By.CSS_SELECTOR, locators.DRAFT_CARD).is_displayed(), is_true())
+        check_that("Published card is displayed",
+                   utilities.find_element(self.driver, By.CSS_SELECTOR, locators.PUBLISHED_CARD).is_displayed(), is_true())
+
+        # Modify the datetime string to be in same format as datetime.now()
+        draft_date1 = (utilities.find_element(self.driver, By.CSS_SELECTOR, locators.UPLOAD_TIME_ON_DRAFT_CARD).text)
+        draft_date2 = draft_date1.split("GMT")
+        pub_date1 = (utilities.find_element(self.driver, By.CSS_SELECTOR, locators.UPLOAD_TIME_ON_PUBLISHED_CARD).text)
+        pub_date2 = pub_date1.split("GMT")
+        print("Date time now::", self.now)
+        draft_date = datetime.strptime(draft_date2[0].strip(), '%a %b %d %Y %H:%M:%S')
+        draft_date = draft_date.replace(tzinfo=pytz.utc)
+        pub_date = datetime.strptime(pub_date2[0].strip(), '%a %b %d %Y %H:%M:%S')
+        pub_date = pub_date.replace(tzinfo=pytz.utc)
+        print("Type of date is now::",draft_date)
+        lcc.log_info("pub_date::%s" % pub_date)
+        lcc.log_info("draft_date::%s" % draft_date)
+
+        check_that("Draft text displayed on card",
+                   utilities.get_text(self.driver, By.CSS_SELECTOR, locators.DRAFT_CARD_TITLE), equal_to("Draft"))
+        check_that("Published text displayed on card",
+                   utilities.get_text(self.driver, By.CSS_SELECTOR, locators.PUBLISHED_CARD_TITLE), equal_to("Published"))
+        check_that("Draft upload date in expected range", self.check_date_in_range(draft_date), is_true())
+        check_that("Uploaded date for published card in expected range", self.check_date_in_range(pub_date), is_true())
+        check_that("Uploaded date for published card should be less than draft uploaded date", pub_date < draft_date,
```

Comment:
@noopurmath3 I think if you use the less than check in this case, then it would automatically print out the published date and draft date as well in the report, wdyt? or maybe add %s for the published and draft uploaded date in the check? It currently reads:
```
CHECK | Expect Uploaded date for published card should be less than draft uploaded date to be a boolean that is equal to true | Got false                           
```

Reply:
As per our discussion havent used less_than check as it does not compare datetime objects